usaspending_api/search/models/transaction_search.py

```diff
@@ -2,18 +2,10 @@
 from django.db import models
 from django.db.models import Q
 
-from usaspending_api.awards.models import TransactionNormalized, Award
 
-
-class TransactionSearch(models.Model):
-    """
-    Fields in this model have all, with the exception of primary/foreign keys, been made nullable because it
-    is directly populated by the contents of a materialized view. The fields used to create the materialized view
-    may or may not be nullable, but those constraints are not enforced in this table.
-    """
-
-    transaction = models.OneToOneField(TransactionNormalized, on_delete=models.DO_NOTHING, primary_key=True)
-    award = models.ForeignKey(Award, on_delete=models.DO_NOTHING, null=True)
+class AbstractTransactionSearch(models.Model):
+    transaction = models.OneToOneField("awards.TransactionNormalized", on_delete=models.DO_NOTHING, primary_key=True)
+    award = models.ForeignKey("awards.Award", on_delete=models.DO_NOTHING, null=True)
```

Comment:
What does changing the syntax for referencing another model do here?

Reply:
It is called a "lazy reference" for Django. The exact same functionality, but cleans up imports and prevents any cases of circular imports in models.